src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -367,6 +367,7 @@ subject_prefixes:
   - EFO
   - ICD10CM
   - ICD10WHO
+  - icd11.foundation
```

Comment:
## `source_metadata`
- [ ] Any need, benefit, or future desire to add something here for ICD11? @matentzn @hrshdhgd 

There's also a `source_metadata` field in `mondo.sssom.config.yml` with 9 entries in it.

<details><summary>Example</summary>
<p>

```yml
  - from: MONDO
    to: OMIM
    metadata:
      object_source: https://www.omim.org/
      object_source_version: http://www.orphadata.org/data/ORDO/ordo_orphanet_3.1.owl
      mapping_date: "2022-02-11"
      comment: "The latest OMIM is integrated with every Mondo release."
```

</p>
</details> 

But I don't think this is strictly necessary for ICD11 at this time. But I don't know for sure what the benefits of adding an entry here would be. I looked around for references for some of these fields, e.g. `object_source_version`, in `mondo-ingest`, but didn't find any outside of this `mondo.sssom.config.yml`.

Reply:
I dont know, I think this is related to the Mondo repo (not Mondo ingest repo), so you can ignore it for now.